### PR TITLE
Gc/add failed create pod sand box

### PR DESCRIPTION
**Pull request by CariZa**

### CODE REVIEW

File: content/posts/kubernetes/failed-create-pod-sandbox.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Describe pod and see what **node** the pod is meant to be running on:
+
+    $ kubectl describe pod-xxx
+
+Ouput will start with something like this, look for the "Node: " part:
+
+```
+Name:         pod-xxx
+Namespace:    office-updates
```
Comment (Spazzy757) [inline content/posts/kubernetes/failed-create-pod-sandbox.md:74]: Maybe make this namespace more generic

**Review by Spazzy757 — COMMENTED** [review]
Looks awesome I'm sure @ianmiell will be thrilled with this. Just a comment on the example namespace but otherwise if @ianmiell is happen I think this is awesome

**Review by Spazzy757 — APPROVED** [review]
LGTM

**Comment by ianmiell** [comment]
This is indeed great, and good enough to merge right away. Thanks!

If you want more bonus points, it might be worth:

- Googling to see whether the issue can be caused by other things (you might have done this - I put the links I've gone over in the REFERENCED DOCS section - even if they do not add anything).

- Going deeper and looking at the root of the issue in code.

**Pull request merged**